src/bundle/SystemInfo/Collector/IbexaSystemInfoCollector.php

```diff
@@ -228,14 +253,14 @@ private function getStability(): string
         return JsonComposerLockSystemInfoCollector::STABILITIES[$stabilityFlag];
     }
 
-    private function getFirstPackage($packageNames): ?ComposerPackage
+    private static function hasPackage(ComposerSystemInfo $composerInfo, array $packageNames): bool
```

Comment:
I'd rather adjust the name to indicate that we're looking to find **any** of the packages passed as second argument.

(suggestion does not include method calls)

```suggestion
    private static function hasAnyPackage(ComposerSystemInfo $composerInfo, array $packageNames): bool
```

Reply:
Renamed (e118c0d6)